lib/server.go

```diff
@@ -2650,13 +2653,13 @@ func (srv *Server) _startConsensus() {
 		}
 
 		select {
-		case <-srv._getFastHotStuffTransitionCheckInterval():
```

Comment:
Calling _getFastHotStuffTransitionCheckInterval() here recomputes a new wait duration every time the event loop runs, which prevents the transition check from ever running. The bullet proof way to set a timer here is to wait for a precomputed `fastHotStuffConsensusTransitionCheckTime` timestamp